tests/Microsoft.Identity.Test.Unit/pop/CryptoProviderTests.cs

```diff
@@ -19,8 +19,8 @@ public void ValidateCannonicalJwk()
             string jwk = provider.CannonicalPublicKeyJwk;
             JObject jwkObj = JObject.Parse(jwk);
 
-            Assert.IsNotNull(jwkObj["E"]);
-            Assert.IsNotNull(jwkObj["N"]);
+            Assert.IsNotNull(jwkObj["e"]);
```

Comment:
I discovered this. Seems like the JObject would take a key that was case insensitive. But now that is no longer the case. Not sure if breaking change or an actual bug fix. Didn't see it mentioned in the release notes anywhere.

@pmaytak @bgavrilMS

Reply:
afaik Newtonsoft is case-insensitive by default. So this would be a breaking change imo. Yea, the 13.x changelog is a bit sparse, but I guess this change would make sense why they upped the major version. One way to verify could be create a test project and use 12.x and 13.x packages and see what the behavior is. If you can't find any info on this, maybe open an issue in their repo? Otherwise, we might need to set this setting globally somewhere to keep the behavior consistent with how it is now. (Don't think that case-sensitivity should matter but just in case.)

Reply:
There's this issue https://github.com/JamesNK/Newtonsoft.Json/issues/815 that suggests that it's case-insensitive. But also maybe different serialization/deserialization methods have different case sensitivity...